src/stretchy.js

```diff
@@ -315,8 +315,22 @@ const encloseSpan = function(
     return img;
 };
 
+const ruleSpan = function(className: string, options: Options): span {
+    // Get a big square image. The parent span will hide the overflow.
+    const pathNode = new domTree.pathNode('bigRule');
+    const attributes = [
+        ["width", "400em"],
+        ["height", "400em"],
+        ["viewBox", "0 0 400000 400000"],
+        ["preserveAspectRatio", "xMinYMin slice"],
+    ];
+    const svg =  new domTree.svgNode([pathNode], attributes);
+    return buildCommon.makeSpan([className, "hide-tail"], [svg], options);
```

Comment:
How is the height (or width) set for the `ruleSpan`?

Reply:
The `hide-tail` class contains the CSS `overflow: hidden;`. So the height that gets rendered is the  `line.style.height` set by `buildHTML.makeLineSpan()`.

Class `frac-line` has always contained the CSS `width: 100%;`, so the width is being set the way it always has been set.

Reply:
Clarification: My response above pertained to `\frac`, `\overline`, and `\underline`. I did not change `\rule` in this PR, so it is still using a `top-border` to render a line.